codice-itest-impl/src/main/java/org/codice/itest/reporter/TestReporterConfiguration.java

```diff
@@ -34,6 +37,11 @@ public ExitCodeReporter exitCodeReporter(){
         return new ExitCodeReporter();
     }
 
+    @Value("${itest.tests:#{null}}")
```

Comment:
❓ Wouldn't defaulting to `null` here throw an exception on line 43?

Reply:
You are right, I went through and changed that logic to what you guys were currently using. It'll be in the next commit.